### PR TITLE
#162268857-adding-create-record-model

### PR DESCRIPTION
**Pull request by NicholusMuwonge**
## Contains a function to create a record that will be used in the controllers.

@mastula77 @richiem @Rhytah @wasswa-derick @habibwasa @habibsentongo

### CODE REVIEW

**Comment by richien** [comment]
Hi, @NicholusMuwonge I've noticed that some of the Travis builds are failing. Hopefully, these can be easily rectified.

**Comment by NicholusMuwonge** [comment]
> Hi, @NicholusMuwonge I've noticed that some of the Travis builds are failing. Hopefully, these can be easily rectified.

I will work on the tests the moment I finish working on my routes and they're working, that's when I will fix them.

File: api/models/record_model.py [hunk 1]
```diff
@@ -19,7 +19,7 @@ def __init__(
         self.record_title = record_title
         self.record_geolocation = record_geolocation
         self.record_type = record_type
-        self.status = 'Pending'
+        self.status = 'Under Investigation'
         self.record_placement_date = datetime.datetime.now().strftime\
```
Comment (Rhytah) [inline api/models/record_model.py]: I think this method call is redundant because in your table you have catered to this value already

**Review by Rhytah — CHANGES_REQUESTED** [review]

File: api/models/record_model.py (hunk 1 above)
Comment (NicholusMuwonge) [reply api/models/record_model.py]: `okay got it let me work on it.

**Pull request merged**